docs/policies/security_incident_management_policy.md

```diff
@@ -0,0 +1,243 @@
+---
+title: "Secure by Design vulnerability management policy"
+summary: The secure by design policy for security incident management.
+    - Amie Easey
+date: 2024-12-20
+---
+
+# Security incident management policy (Secure by Design)
+
+| Term       | Meaning / Application                                               |
+| ---------- | ------------------------------------------------------------------- |
+| Must       | This term is used to state a mandatory requirement of this policy   |
+| Should     | This term is used to state a recommended requirement of this policy |
+| May        | This term is used to state an optional requirement of this policy   |
+
+## Purpose 
+
+The purpose of this policy is to minimise the immediate and long-term business and reputational impact of incidents that have the potential to affect the confidentiality, integrity, or availability of DfE (Department for Education) systems and / or information to enable DfE to react to incidents in a structured and cohesive manner.
+All DfE impacting security incidents must be reported and managed through the established security incident processes. DfE impacting incidents can include those led by the Government Security Group (GSG), Arm’s Length Bodies (ALBs), and major vulnerabilities which require the incident framework for effective management.
```

Comment:
Do we deal with security incidents that don't impact us? Seems like strange wording for a team who are DfE specific.

Reply:
Good point/spot - I think the first 'DfE impacting' can be taken out but I don't know if the second one needs to stay in as reference to an incident that perhaps didn't start within DfE but impacts us?

Reply:
Would suggest removing first use of it, second use of 'DfE impacting' could be 'Incidents that impact DfE can include those...'